src/main/java/org/openstreetmap/atlas/checks/base/BaseCheck.java

```diff
@@ -282,7 +288,7 @@ protected boolean acceptPier()
 
     protected void clearFlaggedIdentifiers()
     {
-        this.flaggedIdentifiers.clear();
+        getFlaggedIdentifiers().clear();
```

Comment:
You should still have "this." prefixed to the method.

Reply:
Inside getFlaggedIdentifiers(), "this" keyword is used to set and return the instance variable "flaggedIdentifiers". So, even though different instances are accessing this BaseCheck class, getFlaggedIdentifiers() sets the value to its own instance variable. Please let me know if I am incorrect.

Reply:
I think he meant it should read ```this.getFlaggedIdentifiers()``` since that method is non static and belongs to the instance of the class.

Reply:
Yes you are correct, but in this particular case it is more about readability. "this" keyword is simply referencing the current object, and "this.getFlaggedIdentifiers()" is exactly the same as "getFlaggedIdentifiers()", it is really just for readability. Generally speaking if you are referencing a method or variable in a class you should prefix it with "this" for readability.